grunt/tasks/translate/parseSchema.js

```diff
@@ -1,120 +1,112 @@
 var path = require("path");
 var _ = require("underscore");
 
-module.exports = function (grunt) {
-  
-  grunt.registerTask("_parseSchemaFiles", function () {
-    
+module.exports = function(grunt) {
+
+  grunt.registerTask("_parseSchemaFiles", function() {
+
     var srcPath = grunt.config("sourcedir");
-    
+
     global.translate.schemaData = {};
-    
+
     getSchemaData();
     processGlobals();
     processPluginLocations();
-    
-    
-    
-    function getSchemaData () {
-      [
-        {
-          type: "models",
-          globPattern: "core/schema/*.schema",
-          bowerAttr: false,
-          schemaLabel: "models"
-        },{
-          type: "components",
-          globPattern: "components/*/properties.schema",
-          bowerAttr: "component",
-          schemaLabel: "components"
-        },{
-          type: "extensions",
-          globPattern: "extensions/*/properties.schema",
-          bowerAttr: "targetAttribute",
-          schemaLabel: "extensions"
-        },{
-          type: "menu",
-          globPattern: "menu/*/properties.schema",
-          bowerAttr: "targetAttribute",
-          schemaLabel: "menu"
-        }
-      ].forEach(function (item) {
+
+    function getSchemaData() {
+      [{
+        type: "models",
+        globPattern: "core/schema/*.schema",
+        bowerAttr: false,
+        schemaLabel: "models"
+      }, {
+        type: "components",
+        globPattern: "components/*/properties.schema",
+        bowerAttr: "component",
+        schemaLabel: "components"
+      }, {
+        type: "extensions",
+        globPattern: "extensions/*/properties.schema",
+        bowerAttr: "targetAttribute",
+        schemaLabel: "extensions"
+      }, {
+        type: "menu",
+        globPattern: "menu/*/properties.schema",
+        bowerAttr: "targetAttribute",
+        schemaLabel: "menu"
+      }].forEach(function(item) {
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?